tests/test.py

```diff
@@ -203,6 +204,47 @@ def test_map_user_slice(self):
         # test slicing
         self.assertListEqual([r.map_id for r in self.mu[0:2]], [795627, 795627])
 
+
+class TestLoader(CGTestCase):
```

Comment:
combine the methods of this class; merge `test_incorrect_map_id` with `test_working_map_id` (to a method named `test_loading_map_id`, same with the three `user_id` methods and the two `username` methods.